### DISCUSSION TITLE
the ai text in the redme is a bit too much

### DISCUSSION THREAD

**Discussion by Ulf3000** [runew0lf/RuinedFooocus#125] [General]
what the title says, its so easy to identify it as chatgpt and zero effort

**Comment by runew0lf** [comment]
actually it was claude.ai not chatgpt.

**Reply by Ulf3000** [reply]
whatever, better would be to just explain the differences to foocus instead , this is github .. but you do it your way, just comes off as super artificial

**Reply by runew0lf** [reply]
thanks i shall!
